app/assets/v2/images/metamask.svg

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We'll want to run `svgo` over this before we merge.  If you have `svgo` installed, you can run something like:  `svgo app/assets/v2/images/metamask.svg` or simply run `make compress-images`

Reply:
Will do ^_^